mne/io/meas_info.py

```diff
@@ -1145,6 +1145,12 @@ def create_info(ch_names, sfreq, ch_types=None):
         Channel types. If None, data are assumed to be misc.
         Currently supported fields are "mag", "grad", "eeg", and "misc".
         If str, then all channels are assumed to be of the same type.
+    montage : None | str | Montage | list of Montage
+        A montage containing channel positions. If str or Montage is
+        specified, the channel info will be updated with the channel
+        positions. Default is None. If DigMontage is specified, the
+        digitizer information will be updated. A list of unique montages,
+        can be specifed and applied to the info.
```

Comment:
Whoa... nope, GitHub just displays the old version on the main page for me for some reason! Sorry for the noise.